src/lxc/console.c

```diff
@@ -260,16 +260,21 @@ int lxc_setup_tios(int fd, struct termios *oldtios)
 
 	newtios = *oldtios;
 
-	/* Remove the echo characters and signal reception, the echo
-	 * will be done with master proxying */
-	newtios.c_iflag &= ~(IGNBRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON|IXANY|IXOFF);
-	newtios.c_iflag &= BRKINT;
```

Comment:
Nope, not really nothing of this is documented here very well. But this is a bitwise and assignment meaning (more telling this to myself than you) that only when c_iflag already has BRKINT set to 1 will it be set to 1 otherwise zero.
My reason for changing this is that now we just keep randomly adding flags without a proper reason. ssh gives us a valid argument to set these flags. cfmakeraw turns off fewer options than ssh and I think the ssh settings make more sense. But I'm fine with using cfmakeraw if you'd prefer that.